### PR TITLE
Fix tests for iVar variants

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

File: tests/software/ivar/variants/main.nf [hunk 1]
```diff
@@ -7,35 +7,35 @@ include { IVAR_VARIANTS } from '../../../../software/ivar/variants/main.nf' addP
...
     def dummy = file("${launchDir}/tests/data/dummy/dummy_file.txt", checkIfExists: true)
     def input = []
-    input = [ [ id:'test'], 
+    input = [ [ id:'test'],
               file("${launchDir}/tests/data/bam/test-sc2-artic-v3-sorted-trimmed.bam", checkIfExists: true) ]
     IVAR_VARIANTS ( input, ref, dummy )
 }
 
 workflow test_ivar_variants_with_gff_with_mpileup {
-    params.gff          = "${launchDir}/tests/data/gff/sarscov2/MN908947.3.gff3"
```
Comment (drpatelh) [inline tests/software/ivar/variants/main.nf:32]:
```
Script compilation error
- file : /camp/svc/scratch/babs/patelh/tmp/pytest_workflow_fa484sgb/ivar_variants_no_gff_no_mpileup/tests/software/ivar/variants/main.nf
- cause: Variable `launchDir` already defined in the process scope @ line 35, column 23.
       def ref = file("${launchDir}/tests/data/fasta/sarscov2/MN908947.3.fa", checkIfExists: true)
                         ^

1 error
```

Comment (drpatelh) [reply tests/software/ivar/variants/main.nf:32]: For some reason NF really doesn't like `launchDir` being specified like that. Can't spot anything other than the fact it's being used to define a `params` in the `workflow` scope...but this fix is a good enough workaround for a simple test.

**Comment by drpatelh** [comment]
Conda tests failing are weird and possibly an issue with NF trying to install multiple packages in parallel https://github.com/nextflow-io/nextflow/issues/1819

**Review by MaxUlysse — APPROVED** [review]
LGTM
Conda is failing, but it does look unreliable for some other tools.
2 CI out of 3 works, that's good enough for now I believe

**Pull request merged**